mceiface.cpp

```diff
@@ -0,0 +1,39 @@
+/*
+ * This file was generated by qdbusxml2cpp version 0.8
```

Comment:
It would be better if the .pro could generate those files based on mce.xml
DBUS_INTERFACES seems to be the way https://doc.qt.io/qt-5.9/qtdbus-remotecontrolledcar-controller-controller-pro.html
Don't forget to add them to .gitignore

Reply:
I just copied this from nemo-qml-plugin-systemsettings. Didn't have second thoughts about it.
I will add the appropriate line to the .pro file.

https://git.sailfishos.org/mer-core/nemo-qml-plugin-systemsettings/blob/master/src/src.pro#L12